pkg/adaptor/proxy/proxy.go

```diff
@@ -126,6 +126,7 @@ func (p *agentProxy) dial(ctx context.Context, address string) (net.Conn, error)
 		},
 		retry.Attempts(0),
 		retry.Context(ctx),
+		retry.DelayType(retry.FixedDelay),
```

Comment:
The default delay is 100ms, and I think it causes too frequent retries.

We can set an upper limit for default backoff delays like this.
https://github.com/confidential-containers/cloud-api-adaptor/blob/ade303e0acddcae9aa94bc753866578215587ae7/pkg/adaptor/cloud/ibmcloud-powervs/provider.go#L107-L109

Reply:
Thanks @yoheiueda. I'll try setting MaxDelay and report the results